docs/book/en/src/use_cases/kubernetes/kubernetes/k3s.md

```diff
@@ -0,0 +1,170 @@
+### Integrating WasmEdge with k3s
+
+This guide will help you integrate WasmEdge with k3s, a lightweight Kubernetes distribution for edge computing. Follow the steps below to set up and deploy WasmEdge applications on k3s.
+
+#### Prerequisites
+- A machine with Linux (preferably Ubuntu)
+- Docker installed
+- `k3s` installed
+- `kubectl` command-line tool installed
+- `crun` container runtime installed
+
+#### Step 1: Install k3s
+
+First, install k3s on your machine. You can do this by running the following command:
+
+```sh
+curl -sfL https://get.k3s.io | sh -
+```
+
+This command installs k3s and starts it automatically. To check the status of k3s, run:
+
+```sh
+sudo systemctl status k3s
+```
+
+#### Step 2: Install containerd and crun-wasmedge
+
+WasmEdge requires the `crun` runtime to run WebAssembly applications. Follow these steps to install and configure `containerd` and `crun-wasmedge`.
+
+1. **Install containerd**:
+
+```sh
+sudo apt-get update
+sudo apt-get install -y containerd
+```
+
+2. **Install crun**:
+
+```sh
+sudo apt-get install -y crun
+```
+
+3. **Install WasmEdge**:
+
+Follow the installation instructions from the [WasmEdge documentation](https://wasmedge.org/docs/quick-start/install/). For example:
```

Comment:
This is a dead link, @hydai do you know which document is proper for this?

Reply:
It should be this one: https://wasmedge.org/docs/start/install

But I don't think we want a document here. Because the docs are no longer provided by this repo. They are moved into WasmEdge/docs repo.